testOutput.ps1

```diff
@@ -1,5 +1,5 @@
 $version = $args[0]
-$formats = @("Html", "Dhtml", "Word", "Excel", "JSON")
+$formats = @("Html", "Dhtml", "Word", "Excel", "JSON", "Cucumber")
```

Comment:
Adjusted `testRunnerCmd.cmd` but did not find where in `testRunnerMsBuild.cmd` to make the change.